docker/services.dev.yml

```diff
@@ -1,17 +1,35 @@
 version: '3.7'
 services:
   ##############################################################################################################
-  # rod-licensing gafl_webapp
+  # rod-licensing gafl_websales
   ##############################################################################################################
-  gafl_webapp:
+  gafl_websales:
```

Comment:
You've renamed this one but not the one in services.yml.  Would prefer to keep consistent.

Perhaps `gafl_webapp_public` and `gafl_webapp_internal` would be a more consistent naming system?

Reply:
I retained the term telesales as I think its understood by the business. gafl_webapp and gafl_webapp_telesales are used